multi_neper.py

```diff
@@ -0,0 +1,229 @@
+                if cur_hash:
+                        output_dicts.append(cur_hash)
+
+        return output_dicts
+
+if __name__ == "__main__":
+        parser = argparse.ArgumentParser()
+
+        parser.add_argument("--neper-dir", help="directory containing Neper binaries", default=".")
+        parser.add_argument("--threads", help="number of threads per Neper instance", default="4", type=int)
+        parser.add_argument("--flows", help="number of flows per Neper instance", default="4", type=int)
+        parser.add_argument("--source-port", default="12345", type=int)
+        parser.add_argument("--port", default="12345", type=int)
+        parser.add_argument("--control-port", default="12866", type=int)
+        parser.add_argument("--devices", help="comma-delimited list of links to run Neper on, i.e. eth1,eth2,eth3",
+                            default="eth1")
+        parser.add_argument("--phys-len", default=4294967296)
+        parser.add_argument("--buffer-size", default=4096*120)
+
+        parser.add_argument("-c", "--client", action="store_true")
+        parser.add_argument("--src-ips", required="--client" not in sys.argv and "-c" not in sys.argv,
+                            help="required for Host to install/remove flow-steering rules, comma-delimited list of client IP addresses")
+        parser.add_argument("-H", "--hosts", required=True,
+                            help="comma-delimited list of host IP addresses")
+
+        parser.add_argument("--q-start", default="8", help="starting queue for flow-steering rules", type=int)
+        parser.add_argument("--q-num", default="4", help=("number of queues for flow-steering rules"
+                                                          " (i.e. if q-start=8 and q-num=4, 2"
+                                                          " flow-steering rules each will be"
+                                                          " installed for queues [8-11])"),
+                                                          type=int)
+
+        parser.add_argument("--dry-run", default=False, action="store_true")
+
+        parser.add_argument("-l", "--length", default=10)
+        parser.add_argument("--log", default="WARNING")
+
+        args = parser.parse_args()
+
+        basicConfig(level=args.log.upper())
+
+        devices = args.devices.split(",")
+        hosts = args.hosts.split(",")
+        src_ips = args.src_ips.split(",")
+
+        dev_to_rule = dict()
+        # setup flow_steering rules
+        if not args.client:
+                info("setting up flow-steering rules")
+                # src_ips = args.src_ips.split(",")
+
+                for i, dev in enumerate(devices):
+                        if not args.dry_run:
+                                run_pre_neper_cmds(dev)
+
+                        # TODO flow-steering rules installed in Neper now
+                        # control_port = args.control_port + i
```

Comment:
flow-steering rules are installed in the tcp_stream binary now, remove extraneous comments and functions (`install_flow_steer_rules()` and `del_flow_steer_rules()`)

Reply:
Yeah delete any commented out code or dead code. I still see functions to install flow steering rules.

Please if possible do a pass to delete all dead code to ease the code review as well.